examples/example.ipynb

```
@@ -0,0 +1,430 @@
+                "        app_label = \"tests\"\n",
+                "\n",
+                "class Publisher(models.Model):\n",
+                "    name = models.CharField(max_length=100)\n",
+                "    address = models.TextField(help_text=\"Publisher's official address\")\n",
+                "\n",
+                "    class Meta:\n",
+                "        app_label = \"tests\"\n",
+                "\n",
+                "\n",
+                "class Book(models.Model):\n",
+                "    title = models.CharField(max_length=200)\n",
```

Ruff (0.11.9)

Line too long (94 > 88)

(E501)